lib/account/widgets/profile_modal_body.dart

```diff
@@ -18,8 +18,18 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 class ProfileModalBody extends StatefulWidget {
   final bool anonymous;
   final bool showLogoutDialog;
```

Comment:
some comments on how this constructors are used will go a long way.
`quickSelectMode`, `reloadOnSwitch` etc.

Reply:
All set! This should be good to go now!